tools/lint/lint.py

```diff
@@ -747,8 +747,19 @@ def changed_files(wpt_root):
 
 def lint_paths(kwargs, wpt_root):
     if kwargs.get("paths"):
-        r = os.path.realpath(wpt_root)
-        paths = [os.path.relpath(os.path.realpath(x), r) for x in kwargs["paths"]]
+        #make new list in case there are multiple added directories
+        AllPaths=[]
+        for path in kwargs.get("paths"):
+            # create extension for called directory and merge it with the existing path
+            extension= path+'/'
+            newPath=(str(os.path.realpath(wpt_root))+'/'+ extension)
+            # get all files from new path
+            paths = list(all_filesystem_paths(newPath))
```

Comment:
Do we first want to check if it's a directory?

Reply:
ah my bad, I got rid of my if statement. Putting that back in

Reply:
Although I am not sure if relpath was sufficient or not